ABOUT.md

```diff
@@ -1,9 +1,72 @@
+#### Shogun is accessible
 
-The wealth of ML open-source software allows us to offer bindings to other sophisticated libraries including: [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), [SVMLight](http://svmlight.joachims.org/), [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), [VowpalWabbit](http://www.hunch.net/~vw/), [Tapkee](http://tapkee.lisitsyn.me/), [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. See our [list of integrated external libraries](http://www.shogun-toolbox.org/page/about/contributions).
+* *many languages* (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and *platforms* (Linux/Unix, MacOS and Windows), easy integration with other ML libraries
 
-Shogun got initiated in 1999 by [Soeren Sonnenburg](http://sonnenburgs.de/soeren) and [Gunnar Raetsch](http://www.raetschlab.org/) (that's where the name ShoGun originates from). It is now developed by a much larger Team cf. [website](http://shogun-toolbox.org/page/about/ourteam) and [AUTHORS](http://www.github.com/shogun-toolbox/shogun/wiki/AUTHORS), and would not have been possible without the patches and bug reports by various people. Statistics on Shogun's development activity can be found on [ohloh](https://www.openhub.net/p/shogun).
+#### Shogun is state-of-the-art
+
+* *efficient and competitive* implementation (from standard to cutting edge algorithms), modern software architecture in C++
+* easy combination of *multiple data representations, algorithm classes and general purpose tools* for rapid prototyping of data pipelines
+
+### Shogun is open source
+
+* *free* and *community-based* [ML training and skills](#useshoguntolearn) not just for students
+
+You can try Shogun in our [cloud](https://cloud.shogun.ml/hub/home) (no installation required) or find out more about our [mission](#mission), [features](#features) and [history](#shogunhistory).
+
+
+
+
+
+
+## <a name="mission"></a> Mission
+
+Machine learning is powerful. Within the last two decades, machine learning has emerged from an eccentric academic field to transforming the way we live our everyday life. Its methods influence society and its tools have a billion-dollar impact on global economy. We want to make sure that the underlying machine learning algorithms are transparent and accessible. We believe that ML should not be a privilege of corporations or political power with the money to buy technical knowledge. 
```

Comment:
hm, I don't like putting the abbreviation into this very sentence since it takes out the strength of the statement. suggestion: use machine learning (ML) in the next sentence (see edit)